CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Change Log
 
+# [1.28.0 (September 28th, 2023)](https://github.com/eclipse/eclipse.jdt.ls/milestone/119?closed=1)
+ * enhancement - Update current method snippet and add `static_method` snippet for interface. See [#2862](https://github.com/eclipse-jdtls/eclipse.jdt.ls/pull/2862).
+ * enhancement - Improve the constructor snippet for additional classes in a file. See [#2841](https://github.com/eclipse-jdtls/eclipse.jdt.ls/pull/2841).
+ * enhancement - Return outline/documentSymbols even for `.java` files that are not in a Java project. See [#2809](https://github.com/eclipse-jdtls/eclipse.jdt.ls/issues/2809).
+ * enhancement - Store the completion kinds requested by completion operation. See [#2857](https://github.com/eclipse-jdtls/eclipse.jdt.ls/pull/2857).
+ * enhancement - Report whether the project has Kotlin Gradle files. See [#2859](https://github.com/eclipse-jdtls/eclipse.jdt.ls/pull/2859).
+ * bug fix - Fix `EmptyStackException` in `textDocument/foldingRange`. See [#2865](https://github.com/eclipse-jdtls/eclipse.jdt.ls/issues/2865).
+ * bug fix - Fix renaming of attribute with `@Singular` annotation. See [#2828](https://github.com/eclipse-jdtls/eclipse.jdt.ls/pull/2828).
+ * bug fix - Handle `null` entries in `gradle/checksums.json`. See [#2775](https://github.com/eclipse-jdtls/eclipse.jdt.ls/pull/2775).
```

Comment:
I think users don't care about fixes in the build of an upstream project

Reply:
Right. I'll just put it under "build" like various other things. Easier to exclude when clients decide what's important.